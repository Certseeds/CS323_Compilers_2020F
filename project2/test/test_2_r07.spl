struct Apple
{
  int weight;
  float round;
};
int test_2_r07()
{
  struct Apple aa;
  float weight_test = 1.0;
  aa.weight = aa + 2;
  return 0;
}
