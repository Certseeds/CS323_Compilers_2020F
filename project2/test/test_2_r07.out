Error type 7 at Line 10: binary operation on non-number variables
Error type 5 at Line 10: unmatching type on both sides of assignment
